common/types/schemas/checkpoint.mol

```diff
@@ -0,0 +1,25 @@
+import basic;
+
+table ProposeCount {
+    address: Byte20,
+    count:   Uint32,
+}
+
+vector ProposeCounts <ProposeCount>;
+
+table CheckpointLockCellData {
+    version:           byte,
+    epoch:             Uint64,
+    period:            Uint32,
+    total_period:      Uint32,
```

Comment:
Useless field.

Reply:
total_period may be useless, why others? where can we store epoch and period of this checkpoint

Reply:
Yes, only total_period is useless. I don't know why it references multi lines.